src/CLU/Commands.Common.Authentication/Properties/Resources.resx

```diff
@@ -139,7 +139,7 @@
     <value>The environment name '{0}' is not found.</value>
   </data>
   <data name="ExpiredRefreshToken" xml:space="preserve">
-    <value>Your Microsoft Azure credential in the Windows PowerShell session has expired. Please log in again. In PowerShell, execute Login-AzureRMAccount for Azure Resource Manager cmdlets or Add-AzureAccount for service management cmdlets.</value>
+    <value>Your Microsoft Azure credential in the Windows PowerShell session has expired. Please log in again. </value>
```

Comment:
Windows? should it be just "Your Microsoft Azure credential in current session has expired"